webpack.config.js

```diff
@@ -1,17 +1,11 @@
 
 const NODE_ENV = process.env.NODE_ENV;
```

Comment:
We could, but at least for now, the development and production webpack files are different in a lot of subtle but significant ways. For example, development uses hot middleware + `react-hot`. At some point, production will probably use just a hash for CSS names rather than the BEM class name style.